app/actors/JvmMetricsCollector.java

```diff
@@ -134,21 +129,19 @@ public String toString() {
 
     private Map<String, ExecutorService> createExecutorServiceMap(
             final ActorSystem actorSystem,
-            final Configuration configuration) {
+            final Config configuration) {
         final Map<String, ExecutorService> executorServices = Maps.newHashMap();
 
         // Add the default dispatcher
-        if (configuration.getBoolean("metrics.jvm.dispatchers.includeDefaultDispatcher", true)) {
+        if (configuration.getBoolean("metrics.jvm.dispatchers.includeDefaultDispatcher")) {
```

Comment:
Does this work when turned on? I had it fail here under 2.6

Reply:
Just a reminder for you to take a look at this.